osquery/tables/system/darwin/system_info.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <string>
+
+#include <osquery/sql.h>
+#include <osquery/tables.h>
+
+#include <osquery/logger.h>
+#include <osquery/database.h>
+#include "osquery/database/query.h"
+#include "osquery/sql/sqlite_util.h"
+#include "osquery/core/conversions.h"
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <IOKit/IOKitLib.h>
```

Comment:
I think you can limit the includes to:

```cpp
#include <osquery/database.h>
#include <osquery/tables.h>

#include <CoreFoundation/CoreFoundation.h>
#include <IOKit/IOKitLib.h>
```

Mostly an educated guess though. ;)